packages/types/src/clerk.ts

```diff
@@ -514,6 +514,12 @@ type ClerkOptionsNavigation = ClerkOptionsNavigationFn & {
 export type ClerkOptions = ClerkOptionsNavigation & {
   appearance?: Appearance;
   localization?: LocalizationResource;
+  /**
+   * Navigation
+   */
+  routerPush?: (to: string) => Promise<unknown> | unknown;
+  routerReplace?: (to: string) => Promise<unknown> | unknown;
+  routerDebug?: boolean;
```

Comment:
✂️ Drop the `router*` props since they are added as part of `ClerkOptionsNavigation`